### PR TITLE
Add OpenAPI operation and schema transformer interfaces

### LINKED ISSUE #56022: Add support for an interface for OpenAPI operation transformations
## Background and Motivation

In the new OpenAPI functionality in .NET 9 preview.4 there is a first-class interface for document transformations, [`IOpenApiDocumentTransformer`](https://github.com/dotnet/aspnetcore/blob/main/src/OpenApi/src/Transformers/IOpenApiDocumentTransformer.cs), but there isn't an analogous interface for operations.

It is possible to perform operation transformations by providing a lambda function, but this can quickly get complicated if you want to do things that are non-trivial, use dependencies from DI etc. compared to if you could encapsulate your logic in a class to perform a specific operation like is possible for document transforms.

## Proposed API

```diff
namespace Microsoft.AspNetCore.OpenApi;

+/// <summary>
+/// Represents a transformer that can be used to modify an OpenAPI operation.
+/// </summary>
+public interface IOpenApiOperationTransformer
+{
+    Task TransformAsync(OpenApiOperation operation, OpenApiOperationTransformerContext context, CancellationToken cancellationToken);
+}

public partial sealed class OpenApiOptions
{
+    public OpenApiOptions UseOperationTransformer<[DynamicallyAccessedMembers(DynamicallyAccessedMemberTypes.PublicConstructors)] TTransformerType>()
+        where TTransformerType : IOpenApiOperationTransformer
+    public OpenApiOptions UseOperationTransformer(IOpenApiOperationTransformer transformer)
}
```

These would be implemented the same way the document support is, so that DI works as expected to activate the types, it's AoT-friendly etc.

## Usage Examples

```csharp
services.AddOpenApi("v1", (options) =>
{
    options.UseOperationTransformer<MyOperationTransformer>();
});

public sealed class MyOperationTransformer(MyService someService, IConfiguration configuration) : IOpenApiOperationTransformer
{
    public Task TransformAsync(
        OpenApiOperation operation,
        OpenApiOperationTransformerContext context,
        CancellationToken cancellationToken)
    {
        if (configuration["DoWhatever"] == "true")
        {
            await someService.DoStuff(operation, cancellationToken);
        }
    }
}
```

## Alternative Designs

Make the [code that enumerates the operations](https://github.com/dotnet/aspnetcore/blob/83573c72d12e1c2018805bc5d461d730bcb9dcc4/src/OpenApi/src/Transformers/DelegateOpenApiDocumentTransformer.cs#L32-L50) re-usable to public callers so that if an operation interface isn't desired, then the user can implement a document transformer to easily enumerate the operations to do a transform in their custom implementation of the interface. Maybe a base class that implements `IOpenApiDocumentTransformer` and applies the visitor pattern to allow the user to override methods to apply transforms:

```diff
namespace Microsoft.AspNetCore.OpenApi;

+public abstract class OpenApiOperationTransformer : IOpenApiDocumentTransformer
+{
+    protected OpenApiOperationTransformer();
+
+    public async Task TransformAsync(OpenApiDocument document, OpenApiDocumentTransformerContext context, CancellationToken cancellationToken); // Enumerates the operations and calls TransformOperationAsync() for each
+
+    protected abstract Task TransformOperationAsync(OpenApiOperation operation, OpenApiOperationTransformerContext context, CancellationToken cancellationToken);
}
```

## Risks

Increased complexity of the OpenAPI generator implementation.

### PR DESCRIPTION
**Pull request by martincostello**
# Add OpenAPI operation and schema transformer interfaces

Add interfaces for transforming OpenAPI operations and schemas.

## Description

- Add `IOpenApiOperationTransformer`.
- Add `IOpenApiSchemaTransformer`.
- Rename `Use*Transfomer()` methods to `Add*Transformer()`.

Fixes #56022

### CODE REVIEW

**Comment by martincostello** [comment]
Test failure is #56372.

File: src/OpenApi/src/Transformers/TypeBasedOpenApiSchemaTransformer.cs
Comment (captainsafia) [reply src/OpenApi/src/Transformers/TypeBasedOpenApiSchemaTransformer.cs:25]: What do the benchmarks show about relative perf of activated schema transformers vs. document-based transformers here?

Comment (martincostello) [reply src/OpenApi/src/Transformers/TypeBasedOpenApiSchemaTransformer.cs:25]: I'm having trouble getting them to run - what am I missing? https://github.com/dotnet/aspnetcore/pull/56395#discussion_r1657326456

File: src/OpenApi/perf/Microbenchmarks/TransformersBenchmark.cs
Comment (captainsafia) [reply src/OpenApi/perf/Microbenchmarks/TransformersBenchmark.cs:1]: Ooof -- I usually develop on macOS/Linux so I don't run into the IIS-specific issues with the build. 😅 

One thing I would check is if you have the C++ Components installed in Visual Studio. Then, perhaps try doing a build in the `src/Servers` directory?

I'll try running them locally as well to see what we can get here.

File: src/OpenApi/sample/Program.cs
Comment (captainsafia) [reply src/OpenApi/sample/Program.cs:1]: I think it's helpful to add as an example.

The sample project is getting a little large, particularly since it is used for snapshot testing, so I'm hoping to split the endpoint registrations into separate files to make things a little cleaner.

That shouldn't affect the transformers samples though.

File: src/OpenApi/perf/Microbenchmarks/TransformersBenchmark.cs
Comment (martincostello) [reply src/OpenApi/perf/Microbenchmarks/TransformersBenchmark.cs:1]: I'll try running them from WSL on my laptop.

Comment (martincostello) [reply src/OpenApi/perf/Microbenchmarks/TransformersBenchmark.cs:1]: I'll have to try again another day - on WSL I'm getting issues from Node trying to build Blazor, which the build fails and complains about if it's not been produced.

Comment (martincostello) [reply src/OpenApi/perf/Microbenchmarks/TransformersBenchmark.cs:1]: Got past that hurdle. For some reason, I need to remove `--offline` here to get things to work: https://github.com/dotnet/aspnetcore/blob/564af610ada026f03a048c0dce947ef5452cd728/eng/Npm.Workspace.nodeproj#L33

Comment (martincostello) [reply src/OpenApi/perf/Microbenchmarks/TransformersBenchmark.cs:1]: These brenchmarks are cursed for me 😅. Now I'm getting these errors at runtime:

```
System.Reflection.TargetInvocationException: Exception has been thrown by the target of an invocation.
 ---> System.TypeLoadException: Could not load type 'System.Collections.Generic.OrderedDictionary`2' from assembly 'System.Collections, Version=9.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'.
   at System.Text.Json.Nodes.JsonObject.SetItem(String propertyName, JsonNode value)
   at System.Text.Json.Nodes.JsonNode.set_Item(String propertyName, JsonNode value)
   at Microsoft.AspNetCore.OpenApi.OpenApiSchemaStore..ctor() in /home/martin/coding/dotnet/aspnetcore/src/OpenApi/src/Services/Schemas/OpenApiSchemaStore.cs:line 18
```

Comment (captainsafia) [reply src/OpenApi/perf/Microbenchmarks/TransformersBenchmark.cs:1]: Ugh, this is my bad, Martin. I introduced this break when I replayed on top of the JsonSchemaExporter APIs in STJ. Those take advantage of the new `OrderedDictionary` type in .NET 9. I hadn't run the benchmarks prior to merging this change so I didn't pick it up.

It seems like the issue is with the runtime version that we target currently. Upgrading to a 9.0.0-preview.7 SDK should solve this. I've given the SDK update PR (https://github.com/dotnet/aspnetcore/pull/56421) another kick so that we can get coherent bits working.

Comment (martincostello) [reply src/OpenApi/perf/Microbenchmarks/TransformersBenchmark.cs:1]: Mystery solved! I'll rebase this once that's merged and try again.

Comment (martincostello) [reply src/OpenApi/perf/Microbenchmarks/TransformersBenchmark.cs:1]: @captainsafia They took forever to run through WSL so I'm not sure I really trust the numbers, but this is what I ended up after rebasing to pick up the SDK fix:

---------------

``` ini

BenchmarkDotNet=v0.13.0, OS=ubuntu 22.04
12th Gen Intel Core i7-1270P, 1 CPU, 16 logical and 8 physical cores
.NET SDK=9.0.100-preview.7.24358.6
  [Host]     : .NET 9.0.0 (9.0.24.35702), X64 RyuJIT
  Job-KFILMQ : .NET 9.0.0 (9.0.24.35702), X64 RyuJIT

Server=True  Toolchain=.NET Core 9.0  RunStrategy=Throughput  

```
|                         Method | TransformerCount |           Mean |          Error |         StdDev |         Median |        Op/s |    Gen 0 |    Gen 1 |    Gen 2 |  Allocated |
|------------------------------- |----------------- |---------------:|---------------:|---------------:|---------------:|------------:|---------:|---------:|---------:|-----------:|
|  **ActivatedOperationTransformer** |               **10** |       **8.551 μs** |      **0.5523 μs** |      **1.6022 μs** |       **8.090 μs** | **116,939.071** |   **0.1984** |   **0.0458** |        **-** |       **7 KB** |
| OperationTransformerAsDelegate |               10 |       5.208 μs |      0.2228 μs |      0.6428 μs |       5.174 μs | 192,010.454 |   0.0076 |        - |        - |       6 KB |
|   ActivatedDocumentTransformer |               10 |       5.888 μs |      0.2443 μs |      0.6850 μs |       5.838 μs | 169,826.787 |   0.0076 |        - |        - |       7 KB |
|  DocumentTransformerAsDelegate |               10 |       6.110 μs |      0.4086 μs |      1.1854 μs |       6.056 μs | 163,662.480 |   0.0076 |        - |        - |       6 KB |
|     ActivatedSchemaTransformer |               10 |       6.305 μs |      0.3327 μs |      0.9437 μs |       6.280 μs | 158,594.138 |   0.1678 |   0.0381 |        - |       6 KB |
|    SchemaTransformerAsDelegate |               10 |  79,227.547 μs | 11,964.3220 μs | 35,277.0735 μs |  72,518.108 μs |      12.622 | 484.3750 | 234.3750 |  78.1250 |  66,156 KB |
|  **ActivatedOperationTransformer** |              **100** |      **26.530 μs** |      **1.0569 μs** |      **2.9635 μs** |      **26.577 μs** |  **37,693.012** |   **0.0305** |        **-** |        **-** |      **15 KB** |
| OperationTransformerAsDelegate |              100 |       7.455 μs |      0.5069 μs |      1.4625 μs |       7.167 μs | 134,139.965 |   0.0076 |        - |        - |       6 KB |
|   ActivatedDocumentTransformer |              100 |      11.290 μs |      0.5722 μs |      1.6418 μs |      11.418 μs |  88,574.115 |   0.2289 |   0.0458 |        - |       9 KB |
|  DocumentTransformerAsDelegate |              100 |       8.466 μs |      0.6078 μs |      1.7144 μs |       8.384 μs | 118,118.205 |        - |        - |        - |       6 KB |
|     ActivatedSchemaTransformer |              100 |       6.574 μs |      0.4935 μs |      1.4158 μs |       6.360 μs | 152,115.586 |   0.1678 |   0.0381 |        - |       6 KB |
|    SchemaTransformerAsDelegate |              100 | 159,105.863 μs | 27,022.7997 μs | 79,677.3346 μs | 151,162.694 μs |       6.285 | 687.5000 | 343.7500 | 109.3750 | 132,576 KB |
|  **ActivatedOperationTransformer** |             **1000** |     **215.771 μs** |      **5.0182 μs** |     **13.5669 μs** |     **212.402 μs** |   **4,634.549** |   **2.4414** |   **0.4883** |        **-** |      **92 KB** |
| OperationTransformerAsDelegate |             1000 |      20.230 μs |      1.2532 μs |      3.5551 μs |      19.549 μs |  49,430.322 |   0.1526 |   0.0305 |        - |       6 KB |
|   ActivatedDocumentTransformer |             1000 |      65.576 μs |      1.5338 μs |      4.3262 μs |      65.502 μs |  15,249.425 |   0.7324 |   0.1221 |        - |      30 KB |
|  DocumentTransformerAsDelegate |             1000 |      19.052 μs |      0.7513 μs |      2.1435 μs |      19.012 μs |  52,489.112 |   0.1678 |   0.0305 |        - |       6 KB |
|     ActivatedSchemaTransformer |             1000 |       6.538 μs |      0.6212 μs |      1.6900 μs |       5.873 μs | 152,955.986 |   0.1526 |   0.0305 |        - |       6 KB |
|    SchemaTransformerAsDelegate |             1000 |  92,101.312 μs | 14,297.9695 μs | 41,481.0020 μs |  99,441.431 μs |      10.858 | 531.2500 | 265.6250 |  93.7500 |  73,196 KB |

Comment (captainsafia) [reply src/OpenApi/perf/Microbenchmarks/TransformersBenchmark.cs:1]: Thanks for running these numbers through! Given the incoming changes in https://github.com/dotnet/aspnetcore/pull/56709 and the work started in https://github.com/dotnet/aspnetcore/pull/56599, I wouldn't block these API changes on perf. We can evaluate the perf numbers all-up when both changes are in and see what progress we can make there. Schema transformers are generally going to have a larger footprint than operation/document transformers given the frequency in which they are invoked.

File: src/OpenApi/src/Services/OpenApiDocumentService.cs
Comment (captainsafia) [inline src/OpenApi/src/Services/OpenApiDocumentService.cs:177]: I think this change violates the ordering principle for transformers that we originally established during API review ([ref](https://github.com/dotnet/aspnetcore/issues/54650#issuecomment-2023317760)) where operation transformers are modeled as document transformers and run in FIFO-order based on their registration. Now it seems like if an operation transformer is an activated instance, it'll run in order with document transformers but if it is a delegate or instantiated then it will run before document transformers.

Assuming my assesssment here is correct, we'll probably want to figure out a way to fix this.

File: src/OpenApi/perf/Microbenchmarks/TransformersBenchmark.cs
Comment (martincostello) [reply src/OpenApi/perf/Microbenchmarks/TransformersBenchmark.cs:1]: Ok cool, thanks for taking a look. I'll sort out the example tomorrow or Friday and then move this out of draft 👍

Comment (captainsafia) [reply src/OpenApi/perf/Microbenchmarks/TransformersBenchmark.cs:1]: Sounds good! I'll hold off merging the other schema transformers PR until this one is in to avoid giving you a rebase headache.

File: src/OpenApi/sample/Program.cs
Comment (martincostello) [reply src/OpenApi/sample/Program.cs:1]: I've added two examples of using transformers.

File: src/OpenApi/src/Services/OpenApiDocumentService.cs
Comment (martincostello) [reply src/OpenApi/src/Services/OpenApiDocumentService.cs:187]: I've added a test for operations running FIFO before the document transformers and refactored things so it passes. The trade-off here was that it's one activation per operation now if the dependency is transient, rather than being created once per document.

File: src/OpenApi/perf/Microbenchmarks/TransformersBenchmark.cs
Comment (captainsafia) [inline src/OpenApi/perf/Microbenchmarks/TransformersBenchmark.cs:82]: We should provide a `Todo` parameter here so the schema transformer paths actually get executed.

**Review by captainsafia — COMMENTED** [review]
Looking good! Some minor comments on the docs and feedback on our benchmark test scenarios.

WRT to perf, we can reasses impact for schema transformers once this PR and https://github.com/dotnet/aspnetcore/pull/56709 are in, but figured we should establish good baselines here.

File: src/OpenApi/perf/Microbenchmarks/TransformersBenchmark.cs
Comment (captainsafia) [reply src/OpenApi/perf/Microbenchmarks/TransformersBenchmark.cs:1]: Welp -- I had written up this comment alongside my earlier review and then GitHub played me and didn't post it. Let me know if you are seeing similar numbers after making the change to the ActivatedTransformer scenarios.

Comment (martincostello) [reply src/OpenApi/perf/Microbenchmarks/TransformersBenchmark.cs:1]: Here's what I get in WSL with the latest changes:

-------------------

``` ini

BenchmarkDotNet=v0.13.0, OS=ubuntu 22.04
12th Gen Intel Core i7-1270P, 1 CPU, 16 logical and 8 physical cores
.NET SDK=9.0.100-preview.7.24358.6
  [Host]     : .NET 9.0.0 (9.0.24.35702), X64 RyuJIT
  Job-RWQJNC : .NET 9.0.0 (9.0.24.35702), X64 RyuJIT

Server=True  Toolchain=.NET Core 9.0  RunStrategy=Throughput  

```
|                         Method | TransformerCount |       Mean |     Error |     StdDev |     Median |      Op/s |  Gen 0 |  Gen 1 | Gen 2 | Allocated |
|------------------------------- |----------------- |-----------:|----------:|-----------:|-----------:|----------:|-------:|-------:|------:|----------:|
|  **ActivatedOperationTransformer** |               **10** |   **6.511 μs** | **0.2847 μs** |  **0.8167 μs** |   **6.520 μs** | **153,594.6** | **0.1755** | **0.0381** |     **-** |      **7 KB** |
| OperationTransformerAsDelegate |               10 |   5.382 μs | 0.3163 μs |  0.8871 μs |   5.228 μs | 185,804.4 | 0.0076 |      - |     - |      6 KB |
|   ActivatedDocumentTransformer |               10 |   6.532 μs | 0.3649 μs |  1.0703 μs |   6.324 μs | 153,103.4 |      - |      - |     - |      7 KB |
|  DocumentTransformerAsDelegate |               10 |   5.602 μs | 0.2916 μs |  0.8505 μs |   5.667 μs | 178,521.0 | 0.0076 |      - |     - |      6 KB |
|     ActivatedSchemaTransformer |               10 |  71.136 μs | 1.7172 μs |  5.0093 μs |  70.855 μs |  14,057.7 | 1.4648 |      - |     - |     56 KB |
|    SchemaTransformerAsDelegate |               10 |  76.440 μs | 3.2091 μs |  9.2590 μs |  75.209 μs |  13,082.1 | 1.4648 |      - |     - |     55 KB |
|  **ActivatedOperationTransformer** |              **100** |  **10.824 μs** | **0.9000 μs** |  **2.4789 μs** |  **10.920 μs** |  **92,389.4** | **0.2289** | **0.0458** |     **-** |      **9 KB** |
| OperationTransformerAsDelegate |              100 |   8.029 μs | 0.4858 μs |  1.3939 μs |   7.804 μs | 124,550.7 |      - |      - |     - |      6 KB |
|   ActivatedDocumentTransformer |              100 |   9.658 μs | 0.5128 μs |  1.4712 μs |   9.669 μs | 103,541.0 | 0.2365 | 0.0534 |     - |      9 KB |
|  DocumentTransformerAsDelegate |              100 |   7.948 μs | 0.3417 μs |  0.9694 μs |   8.019 μs | 125,824.2 |      - |      - |     - |      6 KB |
|     ActivatedSchemaTransformer |              100 |  77.444 μs | 1.7093 μs |  5.0132 μs |  77.709 μs |  12,912.5 | 0.4883 |      - |     - |     66 KB |
|    SchemaTransformerAsDelegate |              100 |  71.226 μs | 1.7780 μs |  5.2425 μs |  71.005 μs |  14,039.9 | 1.4648 |      - |     - |     61 KB |
|  **ActivatedOperationTransformer** |             **1000** |  **69.732 μs** | **2.8724 μs** |  **8.2875 μs** |  **66.498 μs** |  **14,340.7** | **0.7935** | **0.1831** |     **-** |     **30 KB** |
| OperationTransformerAsDelegate |             1000 |  16.873 μs | 0.8374 μs |  2.4691 μs |  16.685 μs |  59,267.5 |      - |      - |     - |      6 KB |
|   ActivatedDocumentTransformer |             1000 |  66.275 μs | 1.2815 μs |  2.6750 μs |  65.474 μs |  15,088.7 | 0.7324 | 0.1221 |     - |     30 KB |
|  DocumentTransformerAsDelegate |             1000 |  20.716 μs | 1.2154 μs |  3.5455 μs |  19.535 μs |  48,272.1 | 0.1526 | 0.0305 |     - |      6 KB |
|     ActivatedSchemaTransformer |             1000 | 262.090 μs | 8.0333 μs | 23.4336 μs | 257.450 μs |   3,815.5 | 3.9063 |      - |     - |    164 KB |
|    SchemaTransformerAsDelegate |             1000 | 146.933 μs | 5.4175 μs | 15.8032 μs | 148.187 μs |   6,805.8 | 0.9766 |      - |     - |    117 KB |

**Comment by captainsafia** [comment]
Test failures seem to be an instance of https://github.com/dotnet/aspnetcore/issues/56828 that is affecting a few other PRs. The particular failrue doesn't seem like a flake but it's not impact all PRs so there might be something here...

File: src/OpenApi/perf/Microbenchmarks/TransformersBenchmark.cs
Comment (captainsafia) [reply src/OpenApi/perf/Microbenchmarks/TransformersBenchmark.cs:1]: Did another run on my end against 28acf62c6f5d081fad40d7b0539bcf0b2cf81b17. The numbers were seeing have some variance but tell the same story about the cost of activation, particularly for schema transformers which get called more frequently in the pipeline.

|                         Method | TransformerCount |       Mean |     Error |    StdDev |      Op/s |  Gen 0 |  Gen 1 | Gen 2 | Allocated |
|------------------------------- |----------------- |-----------:|----------:|----------:|----------:|-------:|-------:|------:|----------:|
|  **ActivatedOperationTransformer** |               **10** |   **2.466 μs** | **0.0377 μs** | **0.0477 μs** | **405,514.3** | **0.0534** | **0.0114** |     **-** |      **7 KB** |
| OperationTransformerAsDelegate |               10 |   2.561 μs | 0.0195 μs | 0.0173 μs | 390,494.5 | 0.0496 | 0.0114 |     - |      6 KB |
|   ActivatedDocumentTransformer |               10 |   2.462 μs | 0.0369 μs | 0.0288 μs | 406,209.9 | 0.0534 | 0.0114 |     - |      7 KB |
|  DocumentTransformerAsDelegate |               10 |   2.399 μs | 0.0463 μs | 0.0455 μs | 416,841.0 | 0.0496 | 0.0114 |     - |      6 KB |
|     ActivatedSchemaTransformer |               10 |  31.617 μs | 0.4569 μs | 0.4274 μs |  31,628.4 | 0.3662 |      - |     - |     56 KB |
|    SchemaTransformerAsDelegate |               10 |  30.972 μs | 0.6155 μs | 0.6320 μs |  32,287.4 | 0.3662 |      - |     - |     55 KB |
|  **ActivatedOperationTransformer** |              **100** |   **4.082 μs** | **0.0557 μs** | **0.0494 μs** | **244,954.8** | **0.0687** | **0.0153** |     **-** |      **9 KB** |
| OperationTransformerAsDelegate |              100 |   3.250 μs | 0.0595 μs | 0.0497 μs | 307,655.7 | 0.0458 | 0.0076 |     - |      6 KB |
|   ActivatedDocumentTransformer |              100 |   4.149 μs | 0.0428 μs | 0.0358 μs | 241,048.5 | 0.0687 | 0.0153 |     - |      9 KB |
|  DocumentTransformerAsDelegate |              100 |   3.241 μs | 0.0258 μs | 0.0215 μs | 308,574.1 | 0.0496 | 0.0114 |     - |      6 KB |
|     ActivatedSchemaTransformer |              100 |  38.633 μs | 0.2470 μs | 0.2190 μs |  25,884.8 | 0.4883 |      - |     - |     66 KB |
|    SchemaTransformerAsDelegate |              100 |  35.940 μs | 0.4064 μs | 0.3802 μs |  27,824.2 | 0.4883 |      - |     - |     62 KB |
|  **ActivatedOperationTransformer** |             **1000** |  **34.187 μs** | **0.4856 μs** | **0.4543 μs** |  **29,251.0** | **0.2441** |      **-** |     **-** |     **30 KB** |
| OperationTransformerAsDelegate |             1000 |  11.128 μs | 0.0788 μs | 0.0658 μs |  89,863.8 | 0.0458 | 0.0153 |     - |      6 KB |
|   ActivatedDocumentTransformer |             1000 |  36.648 μs | 0.6589 μs | 0.8332 μs |  27,286.5 | 0.2441 |      - |     - |     30 KB |
|  DocumentTransformerAsDelegate |             1000 |  12.282 μs | 0.0807 μs | 0.0674 μs |  81,422.7 | 0.0458 | 0.0153 |     - |      6 KB |
|     ActivatedSchemaTransformer |             1000 | 145.059 μs | 1.1951 μs | 0.9980 μs |   6,893.7 | 1.2207 | 0.2441 |     - |    164 KB |
|    SchemaTransformerAsDelegate |             1000 |  72.551 μs | 0.4466 μs | 0.3959 μs |  13,783.4 | 0.7324 |      - |     - |    117 KB |

Comment (captainsafia) [reply src/OpenApi/perf/Microbenchmarks/TransformersBenchmark.cs:1]: See https://github.com/dotnet/aspnetcore/issues/56829.

**Review by captainsafia — APPROVED** [review]

**Review by captainsafia — APPROVED** [review]

**Pull request merged**
